### PR TITLE
Bonemeal on sugarcane, cacti and nether wart

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Everything is configurable. Enabling and disabling requires a restart as it requires reloading of the mixins.

### CODE REVIEW

**Comment by redcreeper14385** [comment]
This is.good, ill be reviewing this and merging ASAP.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ jobs:
   build:
     strategy:
       matrix:
-        java-version: [1.8, 11, 14]
+        java-version: [1.8, 11]
```
Comment (redcreeper14385) [inline .github/workflows/build.yml:7]: I run JDK 14, will removing it here cause issues?

File: src/main/java/mounderfod/moundertweaks/mixin/bonemeal/SugarCaneBlockMixin.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @Shadow
+    @Final
+    public static IntProperty AGE;
+
+    public SugarCaneBlockMixin(Settings settings) {
+        super(settings);
+    }
+
+    @Override
+    public boolean isFertilizable(BlockView world, BlockPos pos, BlockState state, boolean isClient) {
```
Comment (redcreeper14385) [inline src/main/java/mounderfod/moundertweaks/mixin/bonemeal/SugarCaneBlockMixin.java:32]: Is this configurable?

**Review by redcreeper14385 — COMMENTED** [review]
Just a few questions.

File: .github/workflows/build.yml (hunk 1 above)
Comment (BoogieMonster1O1) [reply .github/workflows/build.yml:7]: no

File: src/main/java/mounderfod/moundertweaks/mixin/bonemeal/SugarCaneBlockMixin.java (hunk 2 above)
Comment (BoogieMonster1O1) [reply src/main/java/mounderfod/moundertweaks/mixin/bonemeal/SugarCaneBlockMixin.java:32]: obviously

File: .github/workflows/build.yml (hunk 1 above)
Comment (BoogieMonster1O1) [reply .github/workflows/build.yml:7]: jdk 14 is slow and bad

Comment (redcreeper14385) [reply .github/workflows/build.yml:7]: Whether it's good or not, I still use it but you say it won't affect anything so not a problem.

**Comment by redcreeper14385** [comment]
Looks good, am merging.

**Pull request merged**